README.md

```diff
@@ -1,34 +1,64 @@
 
-```bash
-git clone git@github.com:splunk/<repo slug>.git
-cd <repo dir>
-git submodule update --init --recursive
+## Create File:
 
-#setup python venv must be 3.7
-/Library/Frameworks/Python.framework/Versions/3.7/bin/python3 -m venv .venv
-source .venv/bin/activate
-pip install -r requirements.txt
-pip install https://download.splunk.com/misc/appinspect/splunk-appinspect-latest.tar.gz
+#### URL:
 
-```
+`https://<splunk_ip>:<managment_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/create`
 
+#### Required params:
 
-## Test
+- **file_path**: The path of file along with file name
+- **data**: content to be written in a file
 
-Using docker 
+#### Request Type: POST
 
-```bash
-pytest
-```
+## Delete file:
+
+#### URL:
+
+`https://<splunk_ip>:<managment_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/delete`
+
+#### Required params:
+
+- **file_path**: The path of file which want to delete along with file name
+
+#### Request Type: DELETE
+
+## Read file:
+
+#### URL:
 
-Using external Splunk instance with Eventgen and app pre-installed
+`https://<splunk_ip>:<managment_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/read`
 
-```bash
-pytest --splunk_type=external --splunk_host=something --splunk_user=foo --splunk_password=something
+#### Required params:
+
+- **file_path**: The path of file for want to read the data
+
+#### Request Type: GET
+
+> **Note:** The file content is present under entry[0].get("content").get("file_content") while `output_mode` is set as `json`.
+
+Please follow the bellowed python script for better understanding.
+
+```python
+import requests
+
+create_file_url = "https://<splunk_host_ip>:<splunk_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/create"
+payload = {"file_path": FILE_PATH, "data" : "Hello World"}
+response = requests.request("POST", create_file_url, auth=(USERNAME, PASSWORD), data=payload, verify=False)
+
+read_file_url = "https://<splunk_host_ip>:<splunk_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/read"
+payload = {"file_path": FILE_PATH}
+response = requests.request("GET", read_file_url, auth=(USERNAME, PASSWORD), data=payload, verify=False)
+
+delete_file_url = "https://<splunk_host_ip>:<splunk_port>/servicesNS/nobody/Splunk_TA_Modinput_Test/Splunk_TA_Modinput_Test_perform_crd_operation/<entry>/delete"
+payload = {"file_path": FILE_PATH}
+response = requests.request("DELETE", delete_file_url, auth=(USERNAME, PASSWORD), data=payload, verify=False)
 ```
+
+> **Note:** `<entry>` in the URL is Splunk's interpretation way to differentiate the Splunk's default API with custom rest API. It is static and remain same for all the file actions scenarios.
```

Comment:
Can you please update this statement: 'It is static and "should remain the same"'